package.json

```diff
@@ -87,6 +87,7 @@
     "react-markdown": "2.5.0",
     "react-test-renderer": "16.0.0",
     "sass-loader": "6.0.6",
+    "sass-variables-loader": "0.1.3",
```

Comment:
We should have updated `package-lock.json` with this dependency addition. Users running npm 5+ will encounter changes introduced when installing from a fresh clone. The recommended Node version was bumped to the latest LTS in #3294 and will include npm 5+. I suggest installing this latest version.

Reply:
Pushed to master in 17092075f0d873971b287316a5c60045e5e66e80

Reply:
Thank you for decting this problem and submiting a fast fix 👍  Unfortunately I missed the update and did not detected the probelm :(